README.md

```diff
@@ -54,11 +54,11 @@ Windows, Linux and Mac OS X support Mbed CLI. We're keen to learn about your exp
 
 ### Requirements
 
-* **Python** - Mbed CLI is a Python script, so you'll need Python to use it. We test Mbed CLI with [version 2.7.11 of Python](https://www.python.org/downloads/release/python-2711/). It is not compatible with Python 3.
+* **Python** - Mbed CLI is a Python script, so you'll need Python to use it. We test Mbed CLI with [version 2.7.11 of Python](https://www.python.org/downloads/release/python-2711/) and [version 3.6.0 of Python](https://www.python.org/downloads/release/python-360/)
 
 #### Mbed CLI and pip compatibility
 
-<span class="notes">**Note:** Mbed CLI toolchain versions older than 1.5.1 are **not compatible** with `pip` version 10.0 (or newer). Please use latest Mbed CLI with newer version of `pip`. More details in Mbed CLI issue [#657](https://github.com/ARMmbed/mbed-cli/issues/657).</span>
+<span class="notes">**Note:** Mbed CLI versions older than 1.5.1 are **not compatible** with `pip` version 10.0 (or newer). Also Mbed CLI versions older than 1.7.0 support only Python 2. Please use latest Mbed CLI.</span>
```

Comment:
Don't think the 'Also' is needed here.

Reply:
If you want to keep it, you need a comma.